### PR TITLE
Feature/docs

### PR DESCRIPTION
**Pull request by zackt113**
The documentation. The only issue here is the Github action (pkgdown) and the link: https://ubc-mds.github.io/strapvizr/, which is not working. I guess once we fixed the Github action, the link will work. Let me know if anything I could do! Thanks!

### CODE REVIEW

File: vignettes/my-vignette.Rmd [hunk 1]
```diff
@@ -0,0 +1,179 @@
+---
+title: "How to use strapvizr"
+author: "Julien Gordon, Gautham Pughazhendhi, Zack Tang, and Margot Vore"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{How to use strapvizr}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+`strapvizr` is a package for performing bootstrapping of a sample to produce plots and statistics for use in final reports and documents. This notebook shows how you can utilize the strapvizpy package within a project.
```
Comment (voremargot) [inline vignettes/my-vignette.Rmd]: Wrong Name
```suggestion
`strapvizr` is a package for performing bootstrapping of a sample to produce plots and statistics for use in final reports and documents. This notebook shows how you can utilize the strapvizr package within a project.
```

File: vignettes/my-vignette.Rmd [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+ex_data
+```
+
+## 1. Bootstrap
+There are two functions in the bootstrap module, `bootstrap`and `calculate_boot_stats`. These two functions perform the bootstrapping and calculate the relevant statistics.
+
+### 1.1 `bootstrap_distribution`
+**Functionality**
+
+This function performs the bootstrap and returns an array of the results.
```
Comment (voremargot) [inline vignettes/my-vignette.Rmd]: I think they  are lists in R not arrays
```suggestion
This function performs the bootstrap and returns a list of the results.
```

File: vignettes/my-vignette.Rmd [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+```
+
+```{r}
+class(boot_1)
+class(boot_2)
+```
+### 1.2 `calculate_boot_stats`
+**Functionality**
+
+This function performs bootstrapping and returns a dictionary of the sampling distribution statistics.
```
Comment (voremargot) [inline vignettes/my-vignette.Rmd]: We aren't using dictionaries in R
```suggestion
This function performs bootstrapping and returns a named list of the sampling distribution statistics.
```

File: vignettes/my-vignette.Rmd [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+**Function inputs**
+
+- sample : the data that will be bootstrapped
+- rep : the number of repetitions of bootstrapping
+  - this controls the size of the output array
```
Comment (voremargot) [inline vignettes/my-vignette.Rmd]:
```suggestion
  - this controls the size of the outputted list
```

File: vignettes/my-vignette.Rmd [hunk 5]
```diff
@@ -0,0 +1,179 @@
...
+### 1.1 `bootstrap_distribution`
+**Functionality**
+
+This function performs the bootstrap and returns an array of the results.
+
+**Function inputs**
+
+- sample: the data that will be bootstrapped
+- rep: the number of repetitions of bootstrapping
+  - this controls the size of the output array
```
Comment (voremargot) [inline vignettes/my-vignette.Rmd]:
```suggestion
  - this controls the size of the outputted list
```

File: vignettes/my-vignette.Rmd [hunk 6]
```diff
@@ -0,0 +1,179 @@
...
+  - value between 0 and 1
+- seed : we can set this for reproducibility
+- title : title of the histogram
+- y_axis : name of the y axis
+- path: path to where function should be saved
+  - default is NULL which means the plot will not be saved
+
+```{r fig.width=7, fig.height=4}
+# Plot sampling distibution of 1000 sample means at a 95% confidence interval
+plot_1 <- strapvizr::plot_ci(ex_data, rep=1000, level=0.95, seed=123)
```
Comment (voremargot) [inline vignettes/my-vignette.Rmd]:
```suggestion
plot_1 <- strapvizr::plot_ci(ex_data, rep = 1000, level = 0.95, seed = 123)
```

File: vignettes/my-vignette.Rmd [hunk 7]
```diff
@@ -0,0 +1,179 @@
...
+plot_1
+```
+
+```{r fig.width=7, fig.height=4}
+# Plot sampling distibution of 1000 sample means at a 99% confidence interval
+# with a unique title and a bin size of 50
+title <- "Bootstrapped miles/(US) gallon"
+plot_2 <- strapvizr::plot_ci(ex_data, rep=1000, bin_size=50, level=0.99, title=title,
+                   seed=123, estimator = "var")
```
Comment (voremargot) [inline vignettes/my-vignette.Rmd]:
```suggestion
plot_2 <- strapvizr::plot_ci(ex_data, rep = 1000, bin_size = 50, level = 0.99, title = title,
                   seed = 123, estimator = "var")
```

File: vignettes/my-vignette.Rmd [hunk 8]
```diff
@@ -0,0 +1,179 @@
...
+**Function Inputs**
+
+- stat_list : summary statistics produced by the `calculate_boot_stats()` function
+- precision : the precision of the table values
+  - how many decimal places are shown
+- path : can specify a folder path where you want to save the tables
+  - default is NULL which means the tables will not be saved
+
+```{r}
+stat <- calculate_boot_stats(ex_data, 1000, level=0.95, seed=123)
```
Comment (voremargot) [inline vignettes/my-vignette.Rmd]:
```suggestion
stat <- calculate_boot_stats(ex_data, 1000, level = 0.95, seed = 123)
```

**Review by voremargot — COMMENTED** [review]
There were some words  that were specific to python rather than R that I suggested a fix to as well as some formatting changes! Please fix these before we merge

**Comment by gauthampughazhendhi** [comment]
@zackt113 can you kindly check the entire documentation one more time and make it perfect without any errors?

**Comment by zackt113** [comment]
Sorry guys just did a final check on the documentation. Thanks :), @voremargot for detecting those.  Please let me know how it looks. @gauthampughazhendhi

**Review by gauthampughazhendhi — APPROVED** [review]
LGTM

**Pull request merged**